guide/14-deep-learning/address_standerdization_using_model_extension.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Finally, the <strong>tool</strong> will receive a&nbsp;<code>FeatureSet</code>&nbsp;as output from the <strong>extensible function.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Done</p>